pkg/config/node.go

```diff
@@ -32,7 +33,6 @@ type Cluster struct {
 	IngressVirtualRouterID uint8
 	VIPNetmask             int
 	MasterAmount           int64
-	EtcdBackends           string
```

Comment:
Is nothing else using this field anymore? It seems to me like having the masters listed in the cluster struct is nice (though maybe the field name is not fitting).

Reply:
AFAIK no one is using this field anymore. I'll double-check that.
We have the masters list in LB.backends, do you think we should hold it also in cluster struct?